src/main/java/com/google/devtools/build/lib/runtime/TestSummary.java

```diff
@@ -143,6 +146,18 @@ public Builder addFailedLogs(List<Path> failedLogs) {
       return this;
     }
 
+    public Builder addTotalTestCases(int totalTestCases) {
+      checkMutation(totalTestCases);
+      summary.totalTestCases += totalTestCases;
+      return this;
+    }
+
+    public Builder addFailedTestCases(List<TestCase> failedTestCases) {
```

Comment:
Unclear how this method relates to addFailedTestCases (same name, different arguments) below. Which one should be used? Also, our Java compiler complains about these methods not being next to each other. If you keep both, please move this one next to the other one.

Reply:
I removed it - you are right - the existing one is good enough